### ISSUE TITLE
[Feature Request]: Definable quality format presets

### ISSUE THREAD

**Issue by PhuriousGeorge** [tubearchivist/tubearchivist#306]
### Already implemented?

- [X] I have read through the [wiki](https://github.com/tubearchivist/tubearchivist/wiki).
- [X] I understand the [scope](https://github.com/tubearchivist/tubearchivist/wiki/FAQ) of this project and am aware of the [known limitations](https://github.com/tubearchivist/tubearchivist#known-limitations).

## Describe the solution you'd like
Possibility to define multiple quality format presets for quick future use 

## Additional context
Previously discussed in Discord starting [here](https://discord.com/channels/920056098122248193/920060312311463956/999496973679853568)

> something like a text box at the settings page, input the format string plus a name, then to select the format, use a dropdown to select from your presets while still allowing manual entry if desired

### Your help is needed!

- [ ] Yes I can help!

**Comment by bbilly1** [comment]
Yes, a little drop down where you can select your presets you have defined before. That sounds very convenient.

**Comment by bats77** [comment]
A double drop-down list for video quality and audio quality, instead of line of code ; also available on custom settings for a specific channel
